lib/levelup.js

```diff
@@ -210,25 +181,24 @@ LevelUP.prototype.put = function (key_, value_, options, callback) {
   if (maybeError(this, callback)) { return promise }
 
   options = getOptions(options)
-  key = this._codec.encodeKey(key_, options)
-  value = this._codec.encodeValue(value_, options)
 
   this.db.put(key, value, options, function (err) {
     if (err) {
       return dispatchError(self, new WriteError(err), callback)
     }
-    self.emit('put', key_, value_)
+    self.emit('put', key, value)
     if (callback) { callback() }
```

Comment:
we always have a callback here too (should have done this in the promisify branch)

Reply:
Done.